lib/src/components/NxTabs/NxTabs.scss

```diff
@@ -42,26 +41,49 @@
   margin: 0 var(--nx-spacing-4x) -1px 0;
   // subtract border thickness for consistent spacing
   padding: var(--nx-spacing-4x) var(--nx-spacing-6x) calc(var(--nx-spacing-4x) - 2px) var(--nx-spacing-6x);
+  position: relative;
   text-align: center;
   min-width: 80px;
   max-width: 240px;
+  background-color: transparent;
+  z-index: 0;
+
+  &::before {
+    box-sizing: border-box;
+    border-radius: 6px;
+    content: '';
+    height: calc(100% - 8px);
```

Comment:
The height of the focus "box" is `48px` on Kaleido. It's `46px` here.

Reply:
> The height of the focus "box" is `48px` on Kaleido. It's `46px` here.

Thank you for catching that! Changed here: [6135f18](https://github.com/sonatype/sonatype-react-shared-components/pull/988/commits/6135f18a24ea74dcb7b377ae3f7876fbc9a35bdd)